Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -329,7 +330,7 @@ def set_preprocessor(self, preproc):
     def handleNewSignals(self):
         """Reimplemented from OWWidget.handleNewSignals."""
         self._update_class_selection()
-        self.apply()
+        self.commit()
```

Comment:
I agree it's not unreasonable. But just saying that recently in Orange 2, it proved useful to be able to disable automatic recalculation when connecting multiple classifiers.